packages/shell-api/src/helpers.ts

```diff
@@ -224,10 +224,12 @@ export function processDigestPassword(
  * @param verbose
  */
 export async function getPrintableShardStatus(
-  configDB: Database,
+  db: Database,
   verbose: boolean
-): Promise<Document> {
-  const result = {} as any;
+): Promise<ShardingStatusResult> {
+  const result = {} as ShardingStatusResult;
+
+  const configDB = await getConfigDB(db);
```

Comment:
Uh... this is an unintentional breaking change, no? You can't pass a `configDB` object to `sh.status()` explicitly anymore after this

Reply:
Oh right, my bad, somehow I thought the `getConfigDB` was always happening already.

I suppose if `configDB` is passed explicitly then we won't end up doing shardedDataDistribution? or does `getSiblingDb` imply one can get `admin` db from a given `config` db?